libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/ElectronicHorizonParams.java

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.services.android.navigation.v5.navigation;
+
+class ElectronicHorizonParams {
```

Comment:
This will need to be `public` as soon as options are exposed in `MabpboxNavigation` / `MapboxNavigationOptions`, right?

Reply:
fixed